terms.h

```diff
@@ -7,6 +7,10 @@ extern int LINES, COLS;
 extern int LASTLINE;
 #endif
 
+void clear(void);
+void flush_tty();
```

Comment:
This removes `void` in the argument list changing the semantics. Same with `mouse_end`.